### PR TITLE
Add possibility to open gitter chat on site

### PR DESCRIPTION
**Pull request by denar90**
Related to #427 
gitter_chat

### CODE REVIEW

**Comment by scott-w** [comment]
Does this just use Gitter's sidebar integration?

**Comment by denar90** [comment]
Yes. After pressing support link in nav bar chat will be opened, instead of opening it in new browser tab. So user shouldn't leave our page to chat with us.

**Comment by scott-w** [comment]
I'm a fan :+1:

File: src/layouts/layout.jade [hunk 1]
```diff
@@ -72,6 +72,7 @@ html(lang='en', itemscope, itemtype="http://schema.org/WebPage")
       include ../partials/_transifex_config
       script(src="//cdn.transifex.com/live.js" defer)
       script(src="/js/build.js?t=#{Date.now()}" defer)
+      script(src="//sidecar.gitter.im/dist/sidecar.v1.js" async defer)
```
Comment (samccone) [inline src/layouts/layout.jade]: can we load this on demand when someone clicks instead?

**Comment by denar90** [comment]
Added some stuff regarding to @samccone comment.

**Comment by samccone** [comment]
hot :+1:

**Comment by scott-w** [comment]
I'll re-add my 👍 too. Is there anything we need to do before deploying this?

**Comment by denar90** [comment]
@scott-w no 😄 
Thanks guys, merging then 💃

**Pull request merged**